app/assets/stylesheets/helpers/_header.scss

```
@@ -446,19 +447,24 @@
   }
 
   .survey-form-button {
-    @include button(govuk-colour("white"));
     @include bold-19($line-height: 1.1);
+    padding: 10px 15px 5px 15px;
+    margin-right: .5em;
     color: $govuk-brand-colour;
+    background: white;
+    border: 0;
+    border-bottom: solid 2px silver;
```

Worth swapping `silver` with `$govuk-border-colour`? 

`silver` comes out as `rgb(192, 192, 192)` and `$govuk-border-colour` is `rgb(177, 180, 182)`

```suggestion
    border-bottom: solid 2px $govuk-border-colour;
```